msal/tests/Test.MSAL.NET.UIAutomation/XamarinDroidMSALTests.cs

```diff
@@ -103,6 +103,16 @@ public void PromptBehavior_Consent_SelectAccount()
 
         }
 
+        /// <summary>
+        /// B2C aquire token flow with local account
+        /// </summary>
+        [Test]
+        public void B2CLocalAccountAcquireTokenTest()
+        {
+            var labData = LabUserHelper.GetLabResponseWithB2CUser();
```

Comment:
You can call the function directly in the parameter like Line 131. No need for the extra line and the labData variable.

Reply:
yep...thanks. needed to break this apart for testing earlier, so thanks for catching it.

---
In reply to: [233219790](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1403#discussion_r233219790) [](ancestors = 233219790)